### DISCUSSION TITLE
Gene Expression Parameter

### DISCUSSION THREAD

**Discussion by airfan0** [satijalab/seurat#5619] [Q&A]
Hi, 

I am currently working on a software program. We were following the Seurat pipeline parameters as part of the program, and we were wondering what is your parameter for determining that a gene is expressed in a single cell. Not necessarily differentially expressed genes but just that the gene is expressed. 

Thanks,
Amna

**Comment by samuel-marsh** [comment]
Hi,

What do you mean by Seurat determining if gene is expressed?  Do you mean like in `DotPlot` percent expressing?

Best,
Sam

**Comment by airfan0** [comment]
Hi Sam,

I mean like once you've scaled and log transformed the raw counts into normalized counts, what threshold value do you use to confirm that a gene is actually expressed in a cell? For example, I'm assuming that a value of 0.00001 in expression is so close to 0 that we can almost assume it isn't truly being expressed.

If this doesn't make sense, that is not a problem. I was just wondering if there was some type of threshold you use in your pipeline to confirm a gene is expressed in a certain cell.

Thanks,
Amna

**Reply by timoast** [reply]
There's no generally-accepted threshold, this will vary depending on the dataset
